src/appengine/libs/auth.py

```diff
@@ -83,6 +84,7 @@ def _get_iap_key(key_id):
   """Retrieves a public key from the list published by Identity-Aware Proxy,
   re-fetching the key file if necessary.
   """
+  # pylint: disable=missing-timeout
   resp = requests.get('https://www.gstatic.com/iap/verify/public_key')
```

Comment:
nit: Would it be better to add a timeout?

Reply:
This is the existing code, I added this to pass the style check.